spec/spec_helper.rb

```diff
@@ -33,8 +33,8 @@
   config.include GdsApi::TestHelpers::PublishingApiV2
   config.include GovukSchemas::RSpecMatchers
 
-  config.before(:suite) do
-    User.create!(permissions: ["signin"])
```

Comment:
@kevindew @benthorner why was this removed? The tests don't work without a user (`undefined method undefined method organisation_content_id for nil:NilClass`.

Reply:
There was some confusion around it and the usage of seeds - as db:setup will create the user.

It changed around to be a create user before each test to ensure there was always a user, but that didn't really work out as it wasn't the user under test.

I'm fine to have this be a before(:suite) find_or_create on User to create one if it doesn't exist, although I think @benthorner is keener to always used seed user.